evaluation/data/cmc/sorc/regrid_CMCsnow_to1Dvector.f90

```diff
@@ -0,0 +1,349 @@
+program data_regrid
+
+  use netcdf
+  implicit none
+
+  double precision      :: sec_since
+ 
+  character*256         :: source_path
+  character*256         :: destination_path 
+ 
+  character*256         :: source_filename
+  character*256         :: destination_filename
+  character*256         :: weights_filename
+
+  character*19     :: current_date  ! current date
+  character*19     :: since_date = "1970-01-01 00:00:00"
+  character*4      :: yyyy, year
+  character*2      :: mm, dd
+  character*10     :: idate 
+  character*10     :: fv3_grid
+
+  integer, parameter           :: source_lats           = 706
+  integer, parameter           :: source_lons           = 706
+!!!!!!!!!!!!!! C384 grid points !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  integer :: destination_locs, weight_locs
+  integer :: latloc, lonloc, iwt, ndays
+  integer :: i, j, itotal, id, iyear, im, io, ierr
+  integer :: offset_ss
+
+  integer, dimension(2)                       :: start, count
+  real   , dimension(source_lons,source_lats) :: snowd
+  real   , dimension(source_lons,source_lats) :: source_input
+
+  integer,  allocatable :: source_lookup(:), destination_lookup(:)
+  real*8 ,  allocatable :: weights(:)
+  real   ,  allocatable :: cmcSnowDepth(:)
+  real   ,  allocatable :: cmcSnowDepthWT(:)
+
+  real   ,  allocatable :: xcb(:)    ! destination lon
```

Comment:
should name these something better so that it's clear what they are when reading the code. the weights file variable naming is difficult to understand. this should be something like destination_lon